<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>9</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="user"/>
    <widget class="QWidget" name="login">
     <widget class="QLabel" name="csusm_logo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_box_username">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>290</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_box_password">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login_button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>370</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="search">
     <widget class="QLabel" name="start_time_label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Time</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_home">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="start_time_input">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>118</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="end_time_label">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>90</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>End Time</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>110</y>
        <width>118</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="date_label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QTableView" name="available_list">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>481</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="reserveButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>500</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Reserve</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Available Spots</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="confirmation">
     <widget class="QPushButton" name="confirmation_confirm_button">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>350</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmation_back_button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>350</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm parking spot for x time at x?</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="home">
     <widget class="QPushButton" name="logout_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QListView" name="reservations">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>110</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="new_reservation_button">
      <property name="geometry">
       <rect>
        <x>622</x>
        <y>500</y>
        <width>161</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>New Reservation</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
